specs/experimental/op-contracts-manager.md

```diff
@@ -235,13 +234,22 @@ The high level logic of the upgrade method is as follows:
 
 1. The Upgrade Controller Safe will `DELEGATECALL` to the `OPCM.upgrade()` method.
 2. For each `_systemConfig`, the list of addresses in the chain is retrieved.
-3. For each address, a two step upgrade is used where:
-   1. the first upgrade is to an `InitializerResetter` which resets the `initialized` value.
-   1. the implementation is updated to the final address and `upgrade()` is called on that address.
+3. For each address:
+   1. If it is receiving new state variables (only the SystemConfig for Isthmus), a call is made to:
```

Comment:
I don't think its great to leak the implementation details like only sys config for isthmus into the spec. How I would propose structuring this is having a section "Network Upgrades" and then having a subheading for each network upgrade and define the specifics for that upgrade in there